### PR TITLE
Code formatting style

### PR DESCRIPTION
**Pull request by feedbee**
The whole JS code was reformatted according to [Google JavaScript Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml) using [Google Closure Linter](https://developers.google.com/closure/utilities/). Fixed inconsistence in `manifest.json` formatting.

Most of the changes related to indentation, quote type and space behind `if`.

### CODE REVIEW

File: extension/keysocket-gaana.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-		var playbutton = document.querySelector('div.player_wrapper > a.playPause.pause');
-		var pausebutton = document.querySelector('div.player_wrapper > a.playPause.play');
-		if(playbutton)
-			simulateClick(playbutton);
-		else
-			simulateClick(pausebutton);
-	}
-}
\ No newline at end of file
+    if (key == NEXT) {
```
Comment (borismus) [inline extension/keysocket-gaana.js]: Let's use spaces throughout.

File: extension/keysocket-saavn.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-		if(hidebutton != null) {
-			if(hidebutton.id == 'play')
-				simulateClick(pausebutton);
-			else
-				simulateClick(playbutton);			
-		}
-	}
+    if (key == NEXT) {
+        var nextbutton = document.querySelector('button#fwd');
+        simulateClick(nextbutton);
```
Comment (borismus) [inline extension/keysocket-saavn.js]: s/\t/  /g

**Comment by borismus** [comment]
Thanks for this big fix-up!

**Comment by feedbee** [comment]
All JS code indented with 4 spaces. There are no tabs after my commits. Sorry, I don't really understand the points of your comments :-)
```
$ cat extension/*.js | egrep "\t"
$
```

**Comment by borismus** [comment]
Sorry, my mistake! Mis-read the github diff.

**Pull request merged**